### load dataset

In [ ]:
import sys

sys.path.append("../")

from src.data_loader import DataLoader

loader = DataLoader("../data/raw/data_set.csv")
df = loader.load()
loader.show_infos(df)



Shape: (1025, 14)
<class 'pandas.DataFrame'>
RangeIndex: 1025 entries, 0 to 1024
Data columns (total 14 columns):
 #   Column    Non-Null Count  Dtype  
---  ------    --------------  -----  
 0   age       1025 non-null   int64  
 1   sex       1025 non-null   int64  
 2   cp        1025 non-null   int64  
 3   trestbps  1025 non-null   int64  
 4   chol      1025 non-null   int64  
 5   fbs       1025 non-null   int64  
 6   restecg   1025 non-null   int64  
 7   thalach   1025 non-null   int64  
 8   exang     1025 non-null   int64  
 9   oldpeak   1025 non-null   float64
 10  slope     1025 non-null   int64  
 11  ca        1025 non-null   int64  
 12  thal      1025 non-null   int64  
 13  target    1025 non-null   int64  
dtypes: float64(1), int64(13)
memory usage: 112.2 KB


### initial Dataset Overview 

The dataset was successfully loaded from 'data/raw/data_set.csv'

It contains **1025 rows** and **14 columns**,which means it satisfies the subject requirements
 - more than 1000 rows
 - more than 4 variables